styles/utility.css

```diff
@@ -21,6 +21,7 @@ body{
     margin: 0px auto;
     max-width: 800px;
```

Comment:
you have used rem in other places. try to use rem it will help in responsiveness